integration_test/cases/browser/hover_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Wallaby.Integration.Browser.HoverTest do
+  use Wallaby.Integration.SessionCase, async: true
+
+  setup %{session: session} do
+    {:ok, page: visit(session, "hover.html")}
+  end
+
+  describe "hover/2" do
+    test "sends keys to the specified element", %{page: page} do
```

Comment:
Should we change the test name to something more fitting here?

Reply:
Whoops, yeah I had copied the test structure from the "send_keys" tests and forgot to rename. 👍